src/backend/joanie/core/factories.py

```diff
@@ -484,7 +484,12 @@ def courses(self, create, extracted, **kwargs):
             return
 
         for course in extracted:
-            CourseProductRelationFactory(product=self, course=course)
+            course_organization = course.organizations.first()
+            CourseProductRelationFactory(
+                product=self,
+                course=course,
+                organizations=[course_organization] if course_organization else None,
+            )
```

Comment:
We could the whole course organizations.

```suggestion
                organizations=course.organizations,
```